### ISSUE TITLE
gdm.VarImp failes if the sum of I-spline coefficients for the "Geographic" predictor == 0

### ISSUE THREAD

**Issue by JulFrey** [fitzLab-AL/gdm#22]
From line 286 in gdm.variable.importance.R predictors got removed where the I-Spline coefficients == 0. If this is the "Geographic" predictor which is not a column name in the spTable, the whole table gets empty. I don't have the full overview for this rather complex function, but I think it would be enough to check for this case and set geo <- F:

```
  zeroSum <- fullGDM$predictors[which(sumCoeff==0)]
  if(length(zeroSum)>0){
    message(paste0("Sum of I-spline coefficients for predictors(s) ", zeroSum," = 0"))
    Sys.sleep(1)
    message(paste0("Removing predictor(s) ", zeroSum, " and proceeding with permutation testing..."))
    Sys.sleep(1)
    
    for(z in zeroSum){
      ##select variable columns to be removed from original site-pair table
      if(z == "Geographic") { geo <- F} else {
        testVarCols1 <- grep(paste("^s1.", z, "$", sep=""), colnames(spTable))
        testVarCols2 <- grep(paste("^s2.", z, "$", sep=""), colnames(spTable))
        spTable <- spTable[,-c(testVarCols1, testVarCols2)]
      }
    }
  }
```

**Comment by fitzLab-AL** [comment]
Thanks for reporting the bug & suggesting a solution. Should be fixed in version 1.5.0-5.

**Comment by JulFrey** [comment]
Thank you for fixing it so fast!
